pkg/sentry/platform/ptrace/subprocess_arm64.go

```diff
@@ -160,6 +161,13 @@ func enableCpuidFault() {
 
 // appendArchSeccompRules append architecture specific seccomp rules when creating BPF program.
 // Ref attachedThread() for more detail.
-func appendArchSeccompRules(rules []seccomp.RuleSet) []seccomp.RuleSet {
+func appendArchSeccompRules(rules []seccomp.RuleSet, defaultAction linux.BPFAction) []seccomp.RuleSet {
+	rules = append(rules, seccomp.RuleSet{
+		Rules: seccomp.SyscallRules{
+			unix.SYS_GETCPU: {}, // No vsyscall support for SYS_GETCPU on arm64.
```

Comment:
We don't need to do this for non-vsyscall calls. On arm64, this is required to trap vsyscall calls and emulate them in the Sentry.
https://github.com/google/gvisor/blob/master/pkg/sentry/kernel/task_run.go#L304

Reply:
Right now, we don't emulate any vsyscall calls on arm64:
https://github.com/google/gvisor/blob/master/pkg/sentry/syscalls/linux/linux64_arm64.go#L334

Reply:
Since vsyscall is not supported on arm64, I thinks vsyscall emulation on arm64 is not needed, right?

Reply:
yes, you are right.

Reply:
Got it. Thanks!